### PR TITLE
(Feature) add `fuelIndicator` icon

### PR DESCRIPTION
**Pull request by fernandomg**
This PR adds the `fuelIndicator` icon.

<kbd>image</kbd>

Also adds the `.idea` directory to the `.gitignore` file.

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr46--safereactcomponents.review.gnosisdev.com

File: src/dataDisplay/Icon/index.tsx [hunk 1]
```diff
@@ -121,6 +122,7 @@ const icons = {
   eyeOff,
   filledCross,
   fingerPrint,
+  fuelIndicator,
```
Comment (nicosampler) [inline src/dataDisplay/Icon/index.tsx:125]: Could you take a look at the Theme warning?

File: .gitignore [hunk 2]
```diff
@@ -16,6 +16,7 @@ yalc.lock
 /storybook-static
 
 # misc
+/.idea
```
Comment (nicosampler) [inline .gitignore:19]: what is this used for?

Comment (mikheevm) [reply .gitignore:19]: jetbrains ides

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr46--safereactcomponents.review.gnosisdev.com

**Pull request merged**